applications/rag/README.md

```diff
@@ -74,22 +89,27 @@ This filter can auto fetch the templates in your project. Please refer to the fo
 
 #### Verify Setup
 
-1. Verify Kuberay is setup: run `kubectl get pods -n <namespace>` where the namespace is the one set in `workloads.tfvars`. There should be a Ray head and Ray worker pod in `Running` state (prefixed by `example-cluster-kuberay-head-` and `example-cluster-kuberay-worker-workergroup-`).
+Set your namespace from `workloads.tfvars`:
+```
+NAMESPACE=rag
+```
+
+1. Verify Kuberay is setup: run `kubectl get pods -n $NAMESPACE`. There should be a Ray head (and Ray worker pod on GKE Standard only) in `Running` state (prefixed by `example-cluster-kuberay-head-` and `example-cluster-kuberay-worker-workergroup-`).
 
 2. Verify Jupyterhub service is setup:
 
-* Fetch the service IP: `kubectl get services proxy-public -n <namespace> --output jsonpath='{.status.loadBalancer.ingress[0].ip}'`
+* Fetch the service IP: `kubectl get services proxy-public -n $NAMESPACE --output jsonpath='{.status.loadBalancer.ingress[0].ip}'`
 * Go to the IP in a browser which should display the Jupyterlab login UI.
 
 3. Verify the instance `pgvector-instance` exists: `gcloud sql instances list | grep pgvector`
 
 4. Verify the inference server is setup:
-* Set up port forward
+* Start port forwarding
 ```
-kubectl port-forward -n <namespace> deployment/mistral-7b-instruct 8080:8080 &
+kubectl port-forward -n $NAMESPACE deployment/mistral-7b-instruct 8080:8080
 ```
 
-* Try a few prompts:
+* In a new terminal, try a few prompts:
```

Comment:
why?

Reply:
Background processes are not the most user friendly thing. Most people will know how to create to terminals. Even Google Cloud Console provides tabs for this purpose.